File: Sample VB/Form1.vb
Imports Clockwork
Imports System.Net

Public Class Form1

    Private Sub sendMessage_Click(sender As System.Object, e As System.EventArgs) Handles sendMessage.Click
        Try
            Dim api As Clockwork.API = New Clockwork.API(key.Text)
            Dim sms As Clockwork.SMS = New Clockwork.SMS()
            sms.To = number.Text
            sms.Message = message.Text

            Dim result As SMSResult = api.Send(sms)

            If (result.Success) Then
                MessageBox.Show("Sent - ID: " + result.ID)
            Else
                MessageBox.Show("Error: " + result.ErrorMessage)
            End If

        Catch ex As APIException
            ' You'll get an API exception for errors 
            ' such as wrong key
            MessageBox.Show("API Exception: " + ex.Message)
        Catch ex As WebException
            'Web exceptions mean you couldn't reach the mediaburst server
            MessageBox.Show("Web Exception: " + ex.Message)
        Catch ex As ArgumentException
            ' Argument exceptions are thrown for missing parameters,
            ' such as forgetting to set the username
            MessageBox.Show("Argument Exception: " + ex.Message)
        Catch ex As Exception
            ' Something else went wrong, the error message should help
            MessageBox.Show("Unknown Exception: " + ex.Message)
        End Try
    End Sub
End Class

File: Sample CSharp/Form1.cs
using System;
using System.Net;
using System.Windows.Forms;
using Clockwork;

namespace Clockwork.Samples.CSharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void sendMessage_Click(object sender, EventArgs e)
        {
            try
            {
                API api = new API(key.Text);

                SMS sms = new SMS
                {
                    To = to.Text,
                    Message = message.Text
                };

                SMSResult result = api.Send(sms);

                if (result.Success)
                    MessageBox.Show("Sent\nID: " + result.ID);
                else
                    MessageBox.Show("Error: " + result.ErrorMessage);
            }
            catch (APIException ex)
            {
                // You'll get an API exception for errors 
                // such as wrong key
                MessageBox.Show("API Exception: " + ex.Message);
            }
            catch (WebException ex)
            {
                // Web exceptions mean you couldn't reach the mediaburst server
                MessageBox.Show("Web Exception: " + ex.Message);
            }
            catch (ArgumentException ex)
            {
                // Argument exceptions are thrown for missing parameters,
                // such as forgetting to set the username
                MessageBox.Show("Argument Exception: " + ex.Message);
            }
            catch (Exception ex)
            {
                // Something else went wrong, the error message should help
                MessageBox.Show("Unknown Exception: " + ex.Message);
            }
        }
    }
}
